// protobuf.GetGroupMsg.GetGroupMsgReqIdl
syntax = "proto3";

message GetGroupMsgReqIdl {
    string cuid = 1;
    message DataReq {
        int32 width = 1;
        int32 height = 2;
        int32 smallWidth = 3;
        int32 smallHeight = 4;
        message GroupLastId {
            int64 groupId = 1;
            int64 lastMsgId = 2;
        }
        repeated GroupLastId groupMids = 6;
        string gettype = 7;
    }
    DataReq data = 2;
}
